docs/contributing/accessibility/accessibility.mdx

```diff
@@ -0,0 +1,77 @@
+| :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [1.4.3 Contrast minimum](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)<br/><br/>[1.4.11 Non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) | Content should have contrast meeting or exceeding the WCAG AA guidelines:<br/>&nbsp;&nbsp;- For normal text: 4.5:1<br/>&nbsp;&nbsp;- For large text and graphics: 3:1<br/>&nbsp;&nbsp;- Use the [WebAIM contrast checker](https://webaim.org/resources/contrastchecker/) to learn more.<br/>&nbsp;&nbsp;- When using variables from the Component Library, the `text-main` variable is tested on background and `background-alt` variables. All other backgrounds generally use `text-contrast`. |
+| [1.4.13 Content on Hover of focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)                                                                                      | **Bitwarden design best practice:** Interactive elements should always be visible without a user needing to hover or focus that element. This allows content to be more discoverable and thus accessible to users, while also eliminating the complexity necessary to implement such a pattern to meet WCAG criteria.                                                                                                                                                                            |
+| [2.4.7 Focus visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)                                                                                                               | **Bitwarden design best practice:** All interactive elements should have visible focus, hover states, and the correct cursor stying.                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
We probably could move this repeated text: "**Bitwarden design best practice:**" up into the table header next to description:  "Description of Bitwarden Design Best Practice"

Reply:
The other ones are more widely accepted best practices, while the ones marked in that way are practices that we as a company want to adhere to.

Reply:
With that in mind, I think it makes sense to have these as separate designations for how we want to do it, not so much that it is a generally accepted practice outlined by WCAG.